src/main/java/core/basesyntax/service/handler/PurchaseHandler.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax.service.handler;
+
+public class PurchaseHandler implements OperationHandler {
+    @Override
+    public int updateQuantity(int current, int input) {
```

Comment:
your handlers should take as parameter object with field as in file I told you above and perform needed operations on it including saving to storage, changing balance

Reply:
is it neccessary? it works right